src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -471,13 +489,13 @@ def modifyTaskConfiguration(self, taskConf,
         if generator:
             taskConf["SplittingAlgo"] = "EventBased"
             # Adjust totalEvents according to the filter efficiency
-            taskConf["RequestNumEvents"] = int(taskConf.get("RequestNumEvents", 0) / \
-                                               taskConf.get("FilterEfficiency"))
+            taskConf["RequestNumEvents"] = taskConf.get("RequestNumEvents", 0) // \
+                                               taskConf.get("FilterEfficiency")
             taskConf["SizePerEvent"] = taskConf.get("SizePerEvent", self.sizePerEvent) * \
                                        taskConf.get("FilterEfficiency")
 
         if taskConf["EventsPerJob"] is None:
-            taskConf["EventsPerJob"] = int((8.0 * 3600.0) / (taskConf.get("TimePerEvent", self.timePerEvent)))
+            taskConf["EventsPerJob"] = (8.0 * 3600.0) // taskConf.get("TimePerEvent", self.timePerEvent)
```

Comment:
It returns the floor division in a float format, like 8.5 // 4 = 2.0. 
int() guarantees to return a INT floor division, like int(8.5/4.0) = 2
Should I revert the casting then?